lib/danger/scm_source/git_repo.rb

```diff
@@ -4,20 +4,25 @@
 
 module Danger
   class GitRepo
-    attr_accessor :diff, :log
+    attr_accessor :diff, :log, :folder
 
     def diff_for_folder(folder, from: "master", to: "HEAD")
-      repo = Git.open folder
+      self.folder = folder
+      repo = Git.open self.folder
 
       merge_base = repo.merge_base(from, to)
+      ensure_merge_base_exists!(merge_base)
```

Comment:
When I saw this happen, it was actually the `from` argument to `merge_base` that did not exist locally, so I think that we'll already have raised an error by this point?